.github/workflows/slither.yml

```diff
@@ -0,0 +1,39 @@
+name: 'Slither Analysis'
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - develop
+
+jobs:
+  slither-analyze:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      # Workaround to prevent slither-action from trying to install JS deps.
+      # Without this step, it detects the `package.json``, and since there is no
+      # lockfile it defaults `npm install` which fails due to the preinstall
+      # script to enforce pnpm. https://github.com/crytic/slither-action/issues/44#issuecomment-1338183656
+      - name: Remove package.json
+        run: rm packages/contracts-bedrock/package.json
+
+      - name: Run Slither
+        uses: crytic/slither-action@v0.3.1
+        id: slither
+        with:
+          target: packages/contracts-bedrock
+          slither-config: packages/contracts-bedrock/slither.config.json
+          fail-on: config
+          sarif: results.sarif
+          # https://github.com/crytic/slither-action/issues/70#issuecomment-1932948435
+          slither-version: dev-triage-db
+          slither-args: --triage-database packages/contracts-bedrock/slither.db.json
```

Comment:
`slither-version` will be removed once this `--triage-database` feature is incorporated into a released version of slither